net.mkiol.SpeechNote.yaml

```diff
@@ -0,0 +1,428 @@
+        - --disable-xzdec
+        - --disable-lzmadec
+        - --disable-lzmainfo
+        - --disable-lzma-links
+        - --disable-scripts
+        - --disable-doc
+        - --disable-shared
+        - --enable-static
+        - --with-pic
+      sources:
+        - type: archive
+          url: https://downloads.sourceforge.net/lzmautils/xz-5.4.2.tar.gz
+          md5: 4ac4e5da95aa8604a81e32079cb00d42
+
+    - name: mbrola
+      buildsystem: simple
+      build-commands:
+        - make
+        - 'cp Bin/mbrola /app/bin'
+      sources:
+        - type: archive
+          url: https://github.com/numediart/MBROLA/archive/refs/tags/3.3.tar.gz
+          md5: 06993903c7b8d3a8d21cc66cd5a28219
+
+    - name: espeak
+      buildsystem: simple
+      build-commands:
+        - 'mkdir espeak'
+        - 'tar -x -C espeak --strip=1 -f lib/espeak-ng-1.52-patched.tar.gz'
+        - 'cd espeak && ./autogen.sh'
+        - 'cd espeak && ./configure --prefix=/app --with-pic --with-pcaudiolib=no --enable-static --disable-shared --disable-rpath --with-extdict-ru --with-extdict-zh --with-extdict-zhy'
+        - 'cd espeak && make install'
+      sources:
+        - type: archive
+          url: https://github.com/rhasspy/piper/archive/e967bdc1a838023fe3644f8ed7cf6e1e958b42ef.zip
+          md5: c9924cdb6305f6982171d486d5d633e1
+
+    - name: boost
+      buildsystem: simple
+      build-commands:
+        - './bootstrap.sh --prefix=/app --with-libraries=system'
+        - './b2 -j `nproc` install'
+      sources:
+        - type: archive
+          url: https://boostorg.jfrog.io/artifactory/main/release/1.82.0/source/boost_1_82_0.tar.bz2
+          sha256: a6e1ab9b0860e6a2881dd7b21fe9f737a095e5f33a3a874afc6a345228597ee6
+
+    - name: webrtcvad
+      buildsystem: cmake
+      config-opts:
+        - -DCMAKE_BUILD_TYPE=Release
+        - -DCMAKE_POSITION_INDEPENDENT_CODE=ON
+      sources:
+        - type: archive
+          url: https://github.com/webrtc-mirror/webrtc/archive/ac87c8df2780cb12c74942ec8a473718c76cb5b7.zip
+          md5: f1489c137b354594632d260978b283a4
+        - type: patch
+          path: patches/webrtcvad.patch
+
+    - name: libstt
+      buildsystem: simple
+      build-commands:
+        - 'chmod --recursive u+w .'
+        - 'cp ./libstt.so /app/lib'
+        - 'cp ./libkenlm.so /app/lib'
+        - 'cp ./coqui-stt.h /app/include'
+      sources:
+        - type: archive
+          only-arches:
+            - x86_64
+          strip-components: 0
+          url: https://github.com/coqui-ai/STT/releases/download/v1.4.0/native_client.tflite.Linux.tar.xz
+          md5: c160279b0d53ab144d8d83cbbe4dff39
+        - type: archive
+          only-arches:
+            - aarch64
+          strip-components: 0
+          url: https://github.com/coqui-ai/STT/releases/download/v1.4.0/native_client.tflite.linux.aarch64.tar.xz
+          md5: 4dbb4b639255038d4f2a9fbdc2bd26d9
+
+    - name: openfst
+      buildsystem: simple
```

Comment:
this should be buildsystem autotools instead. You can run autoreconf in a `shell` source.

Reply:
Fixed in 6e46e85